build.proj

```diff
@@ -29,6 +29,7 @@
     <GenerateCodeCoverageReportForAll>true</GenerateCodeCoverageReportForAll>
   </PropertyGroup>
   <Import Project="$(ToolsDir)CodeCoverage.targets" Condition="Exists('$(ToolsDir)CodeCoverage.targets')" />
+  <Import Project="$(ToolsDir)PerfTesting.targets" Condition="Exists('$(ToolsDir)PerfTesting.targets')" />
```

Comment:
Sure, I'll add it. I missed it because all the targets under PerfTesting are conditioned by `'$(Performance)'=='true'`